FirebasePerformance/Sources/AppActivity/FPRAppActivityTracker.m

```diff
@@ -160,15 +163,12 @@ - (FIRTrace *)activeTrace {
  *
  * @return true if the platform could prewarm apps on the current device
  */
-+ (BOOL)isPrewarmAvailable {
-  if (![[[GULAppEnvironmentUtil applePlatform] lowercaseString] isEqualToString:@"ios"]) {
-    return NO;
-  }
-  NSString *systemVersion = [GULAppEnvironmentUtil systemVersion];
-  if ([systemVersion length] == 0) {
-    return NO;
+- (BOOL)isPrewarmAvailable {
+  BOOL canPrewarm = NO;
+  if (@available(iOS 13, *)) {
```

Comment:
Is this iOS 15? Curious why we have this for iOS 13 and above?

Reply:
Is this because Prewarm was available in early versions? If so, can you add that to comments as to why we are looking for iOS13 and above here?

Reply:
Also - Have you been able to verify if the check fails for pre iOS13 versions?

Reply:
> Also - Have you been able to verify if the check fails for pre iOS13 versions?

Yes, verified using simulators. 

> Curious why we have this for iOS 13 and above?

Two reasons:
1. Hub uses 13, not sure why, I have sent an email to ask
2. Since we are currently too restrictive in enabling prewarm-detection, I am trying to cover as much ground as possible by letting the prewarm-detection also run on iOS 13 and 14, in case Apple did something like % rollouts on iOS 13 and 14. 

This seemed like a good choice because it is working for hub, and 13 is not too old that env variable checking could break.

Reply:
Michael said their iOS 13 guard was for double-dispatch which had issues on iOS 12. He has the opinion that we don't need to guard `ActivePrewarm` in the first place because if it's iOS < 15 then the environment variable will just never be there.

Reply:
Sounds good. Can you make sure to reflect these learnings as comments above this check? This would be helpful for future references.

Reply:
Added comment. Also, in Michael's doc that he shared with us, it says they have seen 0.1% app starts on iOS 14 showing signs of pre-warming, so possibly Apple did make percentage roll out on iOS 14 to test it. Perhaps that could be a source of why some of our _as doesn't have counters.

Reply:
Thanks for the updates Leo. One nit suggestion would be to rephrase the comment as this "Check for the OS to be 13.x and above since prewarming feature was enabled from that OS version". The intent is for the developer to understand the rationale of why that version check and not to go into deep details.